From 82d3b474a47994b5f939b81469dfcec933e5c364 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Wed, 12 Jul 2017 13:59:44 -0400
Subject: [PATCH 2/5] dispatch: Don't reference glvnd #defines on non-glvnd
 systems

Broke the build on OSX, oops.

Resolves: https://github.com/anholt/libepoxy/issues/132
Signed-off-by: Adam Jackson <ajax@redhat.com>
(cherry picked from commit 91c9ecebd963c7af1f0ef3d1333ca0a723bdd6d4)
---
 src/dispatch_common.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/dispatch_common.c b/src/dispatch_common.c
index bbbf913..a7c2f74 100644
--- a/src/dispatch_common.c
+++ b/src/dispatch_common.c
@@ -597,9 +597,11 @@ epoxy_egl_dlsym(const char *name)
 void *
 epoxy_conservative_glx_dlsym(const char *name, bool exit_if_fails)
 {
+#ifdef GLVND_GLX_LIB
     /* prefer the glvnd library if it exists */
     if (!api.glx_handle)
 	get_dlopen_handle(&api.glx_handle, GLVND_GLX_LIB, false);
+#endif
 
     return do_dlsym(&api.glx_handle, GLX_LIB, name, exit_if_fails);
 }
-- 
2.13.5

